core/templates/domain/state/StateObjectFactory.ts

```diff
@@ -109,6 +111,40 @@ export class State {
     this.writtenTranslations = otherState.writtenTranslations;
     this.nextContentIdIndex = otherState.nextContentIdIndex;
   }
+
+  getRequiredWrittenTranslationsCount(): number {
+    let interactionId = this.interaction.id;
+    const contentIdsToRemove = [];
+
+    let allContentIds = this.writtenTranslations.getAllContentId();
+
+    // As of now we do not delete interaction.hints when a user deletes
+    // interaction, so these hints audio are not counted in checking
```

Comment:
Why is this comment talking about audio when the function is related to written translations?